test/hack/resource/pkg/resourcetypes/eni.go

```diff
@@ -43,95 +43,67 @@ func (e *ENI) Global() bool {
 }
 
 func (e *ENI) GetExpired(ctx context.Context, expirationTime time.Time, excludedClusters []string) (ids []string, err error) {
-	var nextToken *string
-	for {
-		out, err := e.ec2Client.DescribeNetworkInterfaces(ctx, &ec2.DescribeNetworkInterfacesInput{
-			Filters: []ec2types.Filter{
-				{
-					Name:   lo.ToPtr("tag-key"),
-					Values: []string{k8sClusterTag},
-				},
+	enis, err := e.getAllENIs(ctx, &ec2.DescribeNetworkInterfacesInput{
+		Filters: []ec2types.Filter{
+			{
+				Name:   lo.ToPtr("tag-key"),
+				Values: []string{k8sClusterTag},
 			},
-			NextToken: nextToken,
+		},
+	})
+	if err != nil {
+		return nil, err
```

Comment:
Why pass `nil` here instead of `ids`?

Reply:
My thought was that `ids` hadn't changed at this point, so I was thinking it could be clearer to say `nil` instead of `ids.` Based on the other functions (and other resources) I'll change this to `ids` to match those for consistency.